### DISCUSSION TITLE
Fresh mastodon instance returns 503 for all `/api/v1/accounts/search` requests

### DISCUSSION THREAD

**Discussion by paulober** [mastodon/mastodon#30376] [Q&A]
Hi,
i setup my instance single user `mastodon.paulober.eu` completly new. On a new server, new ip, new keys, new ssl certificates, new db, and so on.
Everything works fine except my mastodon client (browser and iOS app) can't find people on other instances. My nginx proxy logs requests like:
```
192.168.1.1 - - [20/May/2024:16:52:46 +0000] "GET /api/v1/accounts/search?limit=1&q=tapbots%40tapbots.social&resolve=1 HTTP/2.0" 503 44 "-" "Ivory/19300 CFNetwork/1496.0.7 Darwin/23.5.0" "-"
```
Any ideas how to solve this mess?

**Comment by mjankowski** [comment]
Did you ever work this out? Was this a temporary issue with the remote server?

**Reply by paulober** [reply]
No I never figured it out and eventually just removed mastodon from my server.
